packages/edit-site/src/components/global-styles/test/use-global-styles-output.js

```diff
@@ -705,25 +705,25 @@ describe( 'global styles renderer', () => {
 			expect(
 				getStylesDeclarations( blockStyles, 'body', true )
 			).toEqual( [
-				'font-family: sans-serif',
 				'--wp--style--root--padding-top: 33px',
 				'--wp--style--root--padding-right: 33px',
 				'--wp--style--root--padding-bottom: 33px',
 				'--wp--style--root--padding-left: 33px',
 				'background-color: var(--wp--preset--color--light-green-cyan)',
+				'font-family: sans-serif',
 			] );
 		} );
 
 		it( 'Should output padding and other properties if useRootPaddingAwareAlignments is disabled', () => {
 			expect(
 				getStylesDeclarations( blockStyles, 'body', false )
 			).toEqual( [
-				'font-family: sans-serif',
 				'background-color: var(--wp--preset--color--light-green-cyan)',
 				'padding-top: 33px',
 				'padding-right: 33px',
 				'padding-bottom: 33px',
 				'padding-left: 33px',
+				'font-family: sans-serif',
```

Comment:
So what determines the order of the output? This seems to match core theme.json settings order, is that it? (I get that this particular change is due to `font-family` now passing through the style engine, just curious about how the ordering otherwise works)

Reply:
Basically the order in which we process them. There's an array of objects over which we loop to process styles:

https://github.com/WordPress/gutenberg/blob/d265f3030480cda0a390208a880a4d8e96e3c409/packages/style-engine/src/styles/index.ts#L9

Reply:
Oh I see, so the order is internal to the style engine.